File: Python/Easy/3461_Check_If_Digits_Are_Equal_in_String_After_Operations_I/solution.py
class Solution:
    def hasSameDigits(self, s: str) -> bool:
        current = s

        while len(current) > 2:
            temp = ""
            for i in range(len(current)-1):
                temp += str((int(current[i]) + int(current[i+1])) % 10)
            current = temp

        return current[0] == current[1]

File: Golang/Easy/3461_Check_If_Digits_Are_Equal_in_String_After_Operations_I/solution.go
package checkifdigitsareequalinstringafteroperationsi

import "strconv"

func hasSameDigits(s string) bool {
	current := s

	for len(current) > 2 {
		temp := ""
		for i := 0; i < len(current)-1; i++ {
			temp += strconv.Itoa((int(current[i]) + int(current[i+1])) % 10)
		}
		current = temp
	}
	return current[0] == current[1]
}
